frontend/src/scenes/surveys/utils.ts

```diff
@@ -5,3 +5,12 @@ const sanitizeConfig = { ADD_ATTR: ['target'] }
 export function sanitizeHTML(html: string): string {
     return DOMPurify.sanitize(html, sanitizeConfig)
 }
+
+export function validateColor(color: string | undefined, fieldName: string): string | undefined {
+    if (!color) {
+        return undefined
+    }
+    // Test if the color value is valid using CSS.supports
+    const isValidColor = CSS.supports('color', color)
```

Comment:
```suggestion
    let isValidColor = CSS.supports('color', color)
    if (!isValidColor) isValidColor = CSS.supports('color', `#${color}`)
```

I think it's worth trying to add a `#` and try again if it fails, either here or elsewhere.
Attention that the given `color` will be mutated in this case, the caller will need to use the new value if it turns out to the a missing # issue

Reply:
created a new function to sanitize the survey's appearance, this way it'll work even without the #